src/coreclr/minipal/Windows/dn-stdio.cpp

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include <Windows.h>
+#include <dn-stdio.h>
+
+int fopen_u16(FILE** stream, const WCHAR* path, const WCHAR* mode)
+{
+    return _wfopen_s(stream, path, mode);
```

Comment:
`_wfopen` is warning by msvc, but `fopen` is not. Is it safely allowed by SDL? Should the shape of wrapper API be `fopen` instead of `fopen_s`?